src/thor/bidirectional_astar.cc

```diff
@@ -1181,7 +1213,6 @@ bool IsBridgingEdgeRestricted(GraphReader& graphreader,
     const auto* edge = tile->directededge(edgeid);
     if (edge == nullptr) {
       throw std::logic_error("Edge pointer was null in IsBridgingEdgeRestricted");
-      return false;
     }
```

Comment:
delete this block completely its redundant. the call to `GraphTile::DirectedEdge` already throws if the edgeid doesnt make sense